README.md

```diff
@@ -1,55 +1,82 @@
 ﻿# GeolocatorService
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE) ![Version](https://img.shields.io/nuget/v/GeolocatorService?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/GeolocatorService?style=flat-square)
+
 A service to get the user's current location, based on Geolocator, suited to UWP or [Uno](https://platform.uno/) apps.
 
 GeolocatorService aims to simplify getting the user's current location and handling most common scenarios, including getting the permission to obtain this location and handling cases where this permission is denied.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-
 ## Getting Started
 
-#### Xamarin / UWP
-Add the GeolocatorService nuget package to your project heads.
-
-To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive nuget package to your project.
-
-#### .Net Mobile / WinUI
-
-Add the GeolocatorService.Uno.WinUI nuget package to your project heads.
-
-To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive.Uno.WinUI nuget package to your project.
-
-### Add the relevant permissions
-
-#### UWP
-
-Add the Location capability in your manifest.
-
-#### iOS
-
-Add the NSLocationWhenInUsageDescription and NSLocationUsageDescription values to your info.plist file. This is the message that is displayed to your user when the permission for their location is requested. For instance,
-
-```
-<key>NSLocationWhenInUseUsageDescription</key>
-<string>TODO NSLocationWhenInUsageDescription</string>
-<key>NSLocationUsageDescription</key>
-<string>TODO NSLocationUsageDescription</string>
-```
-
-#### Android
-
-Add the location permission to your AssemblyInfo.cs file.
-
-```
-[assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
-```
+1. ### Add Package
+
+   #### .Net Mobile / WinUI
+   
+   Add the GeolocatorService.Uno.WinUI nuget package to your project heads.
+   
+   To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive.Uno.WinUI nuget package to your project.
```

Comment:
That's actually very strange. I don't understand why we would have separate Reactive packages. The reactive extensions seem to be independent of the UI implementation so there should be only 1 that's dependent on the Abstraction package.

Reply:
Made a sugestion in commitee mobile .Net.